_posts/2017-10-23-storm-v1.11.13-released.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: StoRM 1.11.13 released
+author: enrico
+summary: Released packages:<br/><span>StoRM Backend 1.11.13</span>
+---
+
+The StoRM Product Team is pleased to announce the release of
+[StoRM 1.11.13][release-notes] that includes the following updated components:
+
+* StoRM Backend v. [1.11.13][backend-rn]
+* YAIM StoRM v. [4.3.10][yaim-rn]
+
+This release provides fixes to some outstanding bugs:
+
+* fixes gemss failure of recall task final status update;
+* fixes pool account mapping failure that happens with lcmaps-plugins-basic versions greater than 1.6.3, by setting storm user as the owner of the gridmap directory;
+* fixes garbage collector removal of SRM ptp requests that are stuck in SRM_IN_PROGRESS after long time: set the amount of seconds needed to consider an IN-PROGRESS ptp as expired by adding/editing `expired.request.ptp.time` variable value (default is 2592000L secs = 1 month) into storm.properties configuration file; 
```

Comment:
- enhances the request garbage collector so that PrepareToPut requests that are stuck in the state SRM_IN_PROGRESS are automatically expired after a configurable amount of time; see the release notes for more details on how to change the default ptp expiration time (1 month)